messaging/messaging/pom.xml

```diff
@@ -60,6 +60,14 @@
       <groupId>org.wildfly.swarm</groupId>
       <artifactId>security</artifactId>
     </dependency>
+    <dependency>
```

Comment:
This JGruoups dependency is only required if users want to use clustering.
Maybe it should not be added to the messaging POM and the user should add it explicitly to its application POM instead...